client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -774,6 +819,12 @@ export default {
           this.selectedAssignee = this.assignableUsers.find(user => user.id === this.segment.relationships.assignee.data.id)
         }
       })
+
+    Promise.resolve(loadAddonComponents('oracle.recommendation.tab'))
```

Comment:
if I get it right, we could use an `await` within loadAddonComponents (see comment below) and then we should not need the Preomise.resolve here anymore (which looks a bit strange)

But if the change leads to issues. let me know and we leave it like it is.

Reply:
Not so sure why but without the Promise.resolve, the asyncComponents the returned array of addons in will be passed to vue as an Promise object. I can at the same time, log the addons object in the `loadAddonComponents.js` file which logs not a Promise but the actual array. If I keep it as it is now, it works. However I think I have implemented all needed changes in the core for this PR besides that.